src/components/elements/Button.tsx

```diff
@@ -0,0 +1,42 @@
+import React, {FC, ButtonHTMLAttributes} from 'react';
+import Button from '@material-ui/core/Button';
+import PropTypes from 'prop-types';
+
+
+interface IButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
```

Comment:
Rename to `ButtonProps` because this interface is more like a object structure definition